frontend/src/components/settings/pages/developer/index.tsx

```diff
@@ -17,21 +17,10 @@ import { useSetting } from '../../../../utils/hooks/useSetting';
 import RemoteDebuggingSettings from '../general/RemoteDebugging';
 
 const installFromZip = () => {
-  window.DeckyPluginLoader.openFilePicker('/home/deck', true).then((val) => {
+  window.DeckyPluginLoader.openFilePicker('/home/deck', true, true, ['zip'], false, false).then((val) => {
```

Comment:
Let's change this from a hardcoded directory to one relative to the current user if possible.

Reply:
Besides using dumb things like opening ., I would resolve the environment variables for respecting the home directory, however I can't find any way to do this frontend side. It's correct that it's not exposed by default on the frontend by default and is only accessible on the backend?

Reply:
It's fixed on the latest commit, just need to be tested.